packages/@okta/vuepress-site/docs/guides/submit-app-prereq/main/index.md

```diff
@@ -95,7 +95,7 @@ The OIN catalog organizes integrations into use cases. From the OIN Manager, you
 
 | Use case | Integration capability |
 | -------- | ---------------------- |
-| [Single Sign-On](https://www.okta.com/integrations/?category=single-sign-on) (most common) | Enables users to access your application from any device with a single entry of their Okta user credentials. This use case is appropriate for Security Assertion Markup Language (SAML) and OpenID Connect (OIDC) integrations. |
+| [Single Sign-On](https://www.okta.com/integrations/?category=single-sign-on) (most common) | Enables users to access your application from any device with a single entry of their Okta user credentials. This use case is automatically assigned to Security Assertion Markup Language (SAML) and OpenID Connect (OIDC) integrations. <br><br> **Note:** You don't need to select this use case because `Single Sign-On` isn't an option in the **App use case** dropdown. |
```

Comment:
```suggestion
| [Single Sign-On](https://www.okta.com/integrations/?category=single-sign-on) (most common) | Enables users to access your application from any device with a single entry of their Okta user credentials. This use case is automatically assigned to Security Assertion Markup Language (SAML) and OpenID Connect (OIDC) integrations. <br><br> **Note:** You don't need to select this use case because `Single Sign-On` isn't an option in the **App use case** dropdown list. |
```

Reply:
I'm not sure if it matters in devdocs as much, but I think the styleguide for product usually has dropdown act as an adjective for the appropriate noun (menu, list, etc). However, it's ok as-is as well.